docs/content/en/_index.html

```diff
@@ -24,11 +24,15 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="https://github.com/pipe-cd/pipecd">
 		GitHub <i class="fab fa-github ml-2 "></i>
 	</a>
+    <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://pipecd.dev/docs-v1.0.x/">
+    Learn about PipeCD v1 <i class="fas fa-arrow-alt-circle-right ml-2"></i>
```

Comment:
Nits, let remove this one now, we will open the link to pipecd v1 docs officially after we prepared more docs contents.